Maple2.Model/Metadata/Server_Table/InstancedFieldTable.cs

```diff
@@ -0,0 +1,6 @@
+using System.Collections.Generic;
+using Maple2.Model.Enum;
+
+namespace Maple2.Model.Metadata;
```

Comment:
namespace would need to be updated to match

Reply:
can't change namespace, it conflicts with https://github.com/AngeloTadeucci/Maple2/blob/server-table/Maple2.Model/Metadata/ServerTableMetadata.cs#L30

Reply:
ah yeah nvm